admin/src/utils/configs.js

```diff
@@ -6,4 +6,6 @@ const configs = {};
   configs[x] = el ? el.getAttribute("content") : process.env[x];
 });
 
+configs.IS_LOCAL_OR_CUSTOM_CLIENT = document.querySelector("meta[name='env:reticulum_server']") ? false : true;
```

Comment:
Ideally we'd have some explicit way of determining whether we are using a local or custom client. For example, maybe the habitat run hook could always injects a specific meta tag like 
`<meta name="env:is_local_or_custom_client" content="false" />` 

Using the `reticulum_server` meta tag as an indirect proxy feels a bit fragile, and is especially confusing with out context.
I'd suggest at least explaining this line with a "why" comment, maybe pointing to a mana page with more explanation about how these env configs work.

Reply:
nit: Maybe this is more of a preference thing, but I find it redundant to have a ternary whose values are just `false` and `true`. It feels clearer just to negate the value you care about instead of using a conditional. Maybe naming the boolean is a better option:
```suggestion
const hasReticulumServerMetaEnvTag = !!document.querySelector("meta[name='env:reticulum_server']");
configs.IS_LOCAL_OR_CUSTOM_CLIENT =  !hasReticulumServerMetaEnvTag;
```